### PR TITLE
feat(chat): redesign mobile and overlay (Issue #618, #98)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

- minor style changes for mobile and overlay messages
- added context menu on tap if mobile
- add avatar to profile sidebar if mobile

Issues:

- Issue: https://github.com/epam/ai-dial-chat/issues/618
- Issue: https://github.com/epam/ai-dial-chat/issues/98

**UI changes**

https://www.figma.com/file/lwL5jneveZczAj5RLcLFH2/DIAL-Chat?type=design&node-id=5613-9394&mode=design&t=UXZMitpQSYuvcam4-4

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix:`, `feat:`, `feature:`, `chore:`, `hotfix:` or `e2e:`
- [x] if contains breaking changes then the pull request name starts with `fix!:`, `feat!:`, `feature!:`, `chore!:`, `hotfix!:` or `e2e!:`
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by ai-dial-actions** [comment]
Hey there and thank you for opening this pull request! 👋🏼

We require pull request titles to follow the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/) and it looks like your proposed title needs to be adjusted.

Details:

```
No release type found in pull request title "resolve conflict". Add a prefix to indicate what kind of release this pull request corresponds to. For reference, see https://www.conventionalcommits.org/

Available types:
 - fix
 - hotfix
 - feat
 - feature
 - chore
```

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-852.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-852.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-852.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-852.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — CHANGES_REQUESTED** [review]
Icon should be `text-secondary` and avatar is displayed instead if exists 
image
See:
- Issue #98

File: apps/chat/src/components/Chatbar/Conversation.tsx [hunk 1]
```diff
@@ -551,9 +551,9 @@ export const ConversationComponent = ({ item: conversation, level }: Props) => {
         <button
           className={classNames(
             'group flex size-full cursor-pointer items-center gap-2 transition-colors duration-200',
-            messageIsStreaming && 'disabled:cursor-not-allowed',
-            isDeleting && 'pr-12',
-            !messageIsStreaming && !isDeleting && 'group-hover:pr-6',
+            messageIsStreaming
+              ? 'disabled:cursor-not-allowed'
+              : 'group-hover:pr-6',
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chatbar/Conversation.tsx]: `disabled:cursor-not-allowed [&:not(:disabled)]:group-hover:pr-6` can be added as static on previous line

File: apps/chat/src/components/Chat/ChatHeader.tsx [hunk 2]
```diff
@@ -178,16 +184,29 @@ export const ChatHeader = ({
...
+                      : conversation.selectedAddons
+                          ?.filter((id) => !model.selectedAddons?.includes(id))
+                          .map((addon) => (
+                            <ModelIcon
+                              key={addon}
+                              entityId={addon}
+                              size={isSmallScreen() ? 20 : 18}
+                              entity={addonsMap[addon]}
+                            />
+                          ))}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ChatHeader.tsx]: `isSmallScreen()` already checked
```suggestion
                    {isSmallScreen()
                      ? !!conversationSelectedAddons.length && (
                          <>
                            <ModelIcon
                              entityId={conversationSelectedAddons[0]}
                              size={20}
                              entity={addonsMap[conversationSelectedAddons[0]]}
                            />
                            <div className="flex size-5 items-center justify-center rounded bg-layer-4 text-[10px] md:size-[18px]">
                              +{conversationSelectedAddons.length - 1}
                            </div>
                          </>
                        )
                      : conversation.selectedAddons
                          ?.filter((id) => !model.selectedAddons?.includes(id))
                          .map((addon) => (
                            <ModelIcon
                              key={addon}
                              entityId={addon}
                              size={18}
                              entity={addonsMap[addon]}
                            />
                          ))}
```

File: apps/chat/src/components/Chat/ChatHeader.tsx [hunk 3]
```diff
@@ -156,7 +162,7 @@ export const ChatHeader = ({
                   <ModelIcon
                     entityId={conversation.model.id}
                     entity={model}
-                    size={18}
+                    size={isSmallScreen() ? 20 : 18}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ChatHeader.tsx]: it makes sense to store `const iconSize = isSmallScreen() ? 20 : 18`  and use this constant in all places

File: apps/chat/src/components/Chat/ChatHeader.tsx [hunk 4]
```diff
@@ -178,16 +184,29 @@ export const ChatHeader = ({
...
+                            </div>
+                          </>
+                        )
+                      : conversation.selectedAddons
+                          ?.filter((id) => !model.selectedAddons?.includes(id))
+                          .map((addon) => (
+                            <ModelIcon
+                              key={addon}
+                              entityId={addon}
+                              size={20}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ChatHeader.tsx]:
```suggestion
                              size={18}
```

Comment (Alexander-Kezik) [reply apps/chat/src/components/Chat/ChatHeader.tsx]: replaced with iconSize

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-852.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Header/User/UserMobile.tsx [hunk 5]
```diff
@@ -19,11 +20,24 @@ import LogOutIcon from '../../../../public/images/icons/log-out.svg';
...
+
   const { data: session } = useSession();
+
   return (
-    <div className=" w-full border-b border-tertiary p-2 text-primary">
+    <div className="w-full border-b border-tertiary p-2 text-primary">
       <div className="flex h-[42px] items-center">
-        <UserIcon className="mx-2" width={18} height={18} />
+        {session?.user?.image ? (
+          <Image
```
Comment (IlyaBondar) [inline apps/chat/src/components/Header/User/UserMobile.tsx]: There was some issues with `next\image` that why we are using `img` in other places (@mikitabut can know details)

Comment (Alexander-Kezik) [reply apps/chat/src/components/Header/User/UserMobile.tsx]: There were some problems with basePath. Reverted to

**Review by IlyaBondar — DISMISSED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-852.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**